04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md

```diff
@@ -0,0 +1,235 @@
+---
+aliases: 
+- vscode
+- visual studio code
```

Comment:
The aliases show that this file could be split up. A lot of it is useful general VSCode knowledge that could live in its own file and be embedded by heading when this guide references it. The more specific CSS references/explanations etc. could be in this file.
That way I would only add `using VSCode for theme development` as alias of this note and `VSCode` for the `Visual studio note`.

If we do it like that, we'll have its own VSCode guide that can be linked to/expanded.